packages/Frontend/src/containers/AppContainer.jsx

```diff
@@ -51,6 +57,7 @@ const mapStateToProps = state => ({
 
 const mapDispatchToProps = dispatch => ({
   logout: () => dispatch(logout()),
+  login: user => dispatch(login(user)),
```

Comment:
This is how I told redux to pass a prop to AppContainer called login and how it relates to redux. Don't concern yourself with it too much, you'll learn redux in time :-).